public/less/_user.less

```diff
@@ -63,6 +63,21 @@
         }
     }
 
+    &-orgs {
```

Comment:
I would prefer that full class name was here `.user-orgs`

Reply:
OK i will change it, but i prefer smaller css selectors. Easier to work with and i see not huge differences to selector like:
```scss
.user {
&.profile {
}
}
```
This is much easier to overwrite/adjust compared to `.user.profile .ui.card .extra.content ul li:not(:last-child)`.

Reply:
Me too but I don't quite like that single class can not be found in less files this way

Reply:
@lafriks 
i will change it after https://github.com/go-gitea/gitea/pull/6919 is merged or closed. makes much easier to create PRs